sdk/core/azure-core/azure/core/pipeline/transport/_aiohttp.py

```diff
@@ -180,7 +180,11 @@ def _get_request_data(self, request):
         """
         if request.files:
             form_data = aiohttp.FormData(request.data or {})
-            for form_file, data in request.files.items():
+            if hasattr(request.files, "items"):
```

Comment:
I would consider checking if this is a `Mapping`. While it is unlikely that anything else gets all the way in here, `items()`  is such a common method name that there may be instances where a non-mapping type has an attribute with that name with different semantics.

Reply:
Make sense